util/src/main/java/io/kubernetes/client/util/WebSocketStreamHandler.java

```diff
@@ -94,27 +93,22 @@ protected void handleMessage(int stream, InputStream inStream) throws IOExceptio
   }
 
   @Override
-  public void failure(Exception ex) {
-    this.error = ex;
+  public void failure(Throwable t) {
+    this.error = t;
   }
 
-  public Exception getError() {
+  public Throwable getError() {
     return this.error;
   }
 
   @Override
   public synchronized void close() {
     if (state != State.CLOSED) {
       state = State.CLOSED;
-      try {
-        if (null != socket) {
-          // code 1000 means "Normal Closure"
-          socket.close(1000, "Triggered client-side terminate");
-          log.debug("Successfully closed socket.");
-        }
-      } catch (IOException ex) {

```

Comment:
why are we removing this catch?

Reply:
the method signature changed, it doesn't throw IOException now